### PR TITLE
Updated Upgrade section with version details

### PR DESCRIPTION
**Pull request by yarunachalam**
Updated Upgrade section with version details

### CODE REVIEW

File: content/neuvector-prime/azure/contents.lr [hunk 1]
```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
    !["Uninstall extension"](./uninstall.png)
 
 
-### Checking the Kubernetes Service Extension version:
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+The existing AKS cluster must be on a supported PAYG platform with its Neuvector configuration. To manually upgrade the recent version via the "az k8s-extension update" command.
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: I don't understand what this is trying to say...  Is this saying that this upgrade only works if they're upgrading an existing NV PAYG deployment?
The second sentence needs to be reworded a bit as well... maybe the following:
Manually upgrading the version of NeuVector will utilize the "az k8s-extension update" command.

File: content/neuvector-prime/azure/contents.lr [hunk 2]
```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
...
+
+Execute the following commands in Cluster Cloud Shell:
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: in a Cluster Cloud Shell

File: content/neuvector-prime/azure/contents.lr [hunk 3]
```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
...
 
 -  ---
    **NOTE** if cluster is deployed in westcentralus location, execute the following command.
            This command will also work on other regions once the feature is rolled out to other regions.
 
 -  ---
 
+To see the most recent published extension version to be upgraded.
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: this will show the latest version available? maybe rephrase as:
"To see the most recently published extension version available as an upgrade."

File: content/neuvector-prime/azure/contents.lr [hunk 4]
```diff
@@ -453,6 +456,12 @@ Versions
 50202.1.202310252
 ```
 
+Execute the following command for extension update:
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: Execute the following command to perform the extension update:

**Review by jeremy-moffitt — CHANGES_REQUESTED** [review]
need to reprhase a few things... I dont understand the first text block and need some explanation.

File: content/neuvector-prime/azure/contents.lr (hunk 1 above)
Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: it was copy from the old upgrade aws doc. Here is the updated

`To Update the NeuVector version to the latest version supported by the marketplace listing, via the "az k8s-extension update" command.              

Execute the following commands in a Cluster Cloud Shell:

-  ---
   **NOTE** if cluster is deployed in westcentralus location, execute the following command.
           This command will also work on other regions once the feature is rolled out to other regions.

-  ---
`

File: content/neuvector-prime/azure/contents.lr [hunk 5]
```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
...
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+To Update the NeuVector version to the latest version supported by the marketplace listing, via the "az k8s-extension update" command.
+
+Execute the following commands in a Cluster Cloud Shell:
 
 -  ---
    **NOTE** if cluster is deployed in westcentralus location, execute the following command.
            This command will also work on other regions once the feature is rolled out to other regions.
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: not part of this change, but "is rolled out to other regions" should be "is rolled out to those regions"

File: content/neuvector-prime/azure/contents.lr [hunk 6]
```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
    !["Uninstall extension"](./uninstall.png)
 
 
-### Checking the Kubernetes Service Extension version:
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+To Update the NeuVector version to the latest version supported by the marketplace listing, via the "az k8s-extension update" command.
+
+Execute the following commands in a Cluster Cloud Shell:
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: this section above needs some more work...  the "To Update..." sentence basically says "to update do" but never tells what to do... Maybe:
"Updating the NeuVector version to the latest version supported by the marketplace listing can be accomplished using the "az k8s-extension update" command.

then update the Execute line to something like:
"The following az k8s-extension commands should be executed in a Cluster Cloud Shell."

Tweak the note to read "NOTE the list-versions command to show the latest published version is available in the westcentralus region and will be available in other regions as the feature is rolled out to those regions".

**Review by jeremy-moffitt — CHANGES_REQUESTED** [review]
I would recommend making some additional changes to the way the description at the top of the section reads.

File: content/neuvector-prime/azure/contents.lr [hunk 7]
```diff
@@ -453,6 +456,12 @@ Versions
 50202.1.202310252
 ```
 
+Execute the following command for the extension update:
```
Comment (jeremy-moffitt) [inline content/neuvector-prime/azure/contents.lr]: Execute the following command to perform the extension update:

File: content/neuvector-prime/azure/contents.lr [hunk 8]
```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
...
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+Updating the NeuVector version to the latest version supported by the marketplace listing can be accomplished using the "az k8s-extension update" command.
+
+The following az k8s-extension commands should be executed in a Cluster Cloud Shell:
 
 -  ---
-   **NOTE** if cluster is deployed in westcentralus location, execute the following command.
-           This command will also work on other regions once the feature is rolled out to other regions.
+   **NOTE:**  if cluster is deployed in westcentralus location, execute the following command to view the available versions.
+            This command will also work on other regions once the feature is rolled out to those regions.
```
Comment (KeithMnemonic) [inline content/neuvector-prime/azure/contents.lr]: s/work on/work in

File: content/neuvector-prime/azure/contents.lr [hunk 9]
```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
    !["Uninstall extension"](./uninstall.png)
 
 
-### Checking the Kubernetes Service Extension version:
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+Updating the NeuVector version to the latest version supported by the marketplace listing can be accomplished using the "az k8s-extension update" command.
+
+The following az k8s-extension commands should be executed in a Cluster Cloud Shell:
```
Comment (KeithMnemonic) [inline content/neuvector-prime/azure/contents.lr]: It does not have to be a Cluster Cloud Shell.  They could run this from anywhere that the az command is installed.

File: content/neuvector-prime/azure/contents.lr [hunk 10]
```diff
@@ -426,16 +426,19 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
    !["Uninstall extension"](./uninstall.png)
 
 
-### Checking the Kubernetes Service Extension version:
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+Updating the NeuVector version to the latest version supported by the marketplace listing can be accomplished using the "az k8s-extension update" command.
```
Comment (KeithMnemonic) [inline content/neuvector-prime/azure/contents.lr]: s/supported by the/available in the/

File: content/neuvector-prime/azure/contents.lr (hunk 9 above)
Comment (KeithMnemonic) [reply content/neuvector-prime/azure/contents.lr]: should it be `az k8s-extension` ?

**Review by KeithMnemonic — CHANGES_REQUESTED** [review]
a few comments.

File: content/neuvector-prime/azure/contents.lr (hunk 9 above)
Comment (yarunachalam) [reply content/neuvector-prime/azure/contents.lr]: @KeithMnemonic , 
The following az k8s-extension commands should be executed in a Cluster Cloud Shell or or Node which has access to Azure CLI:

**Review by KeithMnemonic — APPROVED** [review]
LGTM

File: content/neuvector-prime/azure/contents.lr [hunk 11]
```diff
@@ -426,22 +426,25 @@ The AKS cluster already has the extension. Uninstall the extension as it suggest
    !["Uninstall extension"](./uninstall.png)
 
 
-### Checking the Kubernetes Service Extension version:
+###Upgrading a NeuVector Prime PAYG Cluster
 
-To see the most recent published extension versions, execute the following command in Cloud Shell.
+Updating the NeuVector version to the latest version available in the marketplace listing can be accomplished using the "az k8s-extension update" command.
+
+The following az k8s-extension commands should be executed in a Cluster Cloud Shell or or Node which has access to Azure CLI:
```
Comment (rjschwei) [inline content/neuvector-prime/azure/contents.lr:433]: s/or or/or/
